doc/api.md

```diff
@@ -257,6 +257,8 @@ Additionally for commands (which are attributes of type `command`) the following
     particular IOTAs documentation for allowed values of this field in each case.
 -   **contentType**: `content-type` header used when send command by HTTP transport (ignored in other kinds of
     transports)
+-   **headers**: extra customer headers used when send command by HTTP transport (ignored in other kinds of
```

Comment:
Maybe we should explain that the value of this field is a map which keys are header names and values are jexl expressions? In addition, maybe an example could help.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1715/commits/14dc6b09d6f560faa813122253154bc27917a8b8